.github/workflows/release-tag.yaml

```diff
@@ -0,0 +1,56 @@
+name: release-tag
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
```

Comment:
Do we need this anymore? I don't think we're using the schema submodule anymore. @james-milligan, can you confirm?

Reply:
Removed